tools/templates/acrbuild.yml

```diff
@@ -45,3 +45,4 @@ jobs:
       azureSubscription: azureiiot
       scriptLocation: inlineScript
       inlineScript: powershell ./tools/scripts/acr-build.ps1 $(dockerFolder)
+      
```

Comment:
Did not migrate debug image building due to this change in main -
https://github.com/Azure/Industrial-IoT/commit/9750d55b33f35baf69c9705ef297b12bcce7cb60#diff-7a7e9ac2cd79106fd60115e06f721f070a7b731595080d874deb13a96a93fc37